install/kubernetes/cilium/charts/hubble-ui/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+name: hubble-ui
+version: 0.0.3
+appVersion: 0.0.1
```

Comment:
I know you copied this over from the original chart, but within Cilium these versions must match the other charts. Please run `make -C install/kubernetes` to make sure you use the generated version numbers here.